tools/release/java/src/main/resources/com/ibm/icu/dev/tools/docs/genReport_md.xslt

```diff
@@ -0,0 +1,237 @@
+<!--
+* © 2017 and later: Unicode, Inc. and others.
+* License & terms of use: http://www.unicode.org/copyright.html#License
+-->
+<!--
+/*
+*******************************************************************************
+* Copyright (C) 2016 and later: Unicode, Inc. and others.
+* License & terms of use: http://www.unicode.org/copyright.html
+* Copyright (C) 2008-2013, International Business Machines Corporation and         *
+* others. All Rights Reserved.                                                *
+*******************************************************************************
+* This is the XSLT for the API Report.
+*/
+-->
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+  <xsl:param name="leftVer" />
+  <xsl:param name="rightVer" />
+  <xsl:param name="dateTime" />
+  <xsl:param name="rightMilestone" />
+  <xsl:param name="leftMilestone" />
+  <xsl:param name="notFound" />
+  <xsl:param name="ourYear" />
+  <xsl:output method="text" indent="no" omit-xml-declaration="yes" />
+  <xsl:template match="/">
+<![CDATA[<!--]]>
+ Copyright © <xsl:value-of select="$ourYear" />: Unicode, Inc. and others.
```

Comment:
Should we fix this to 2019 and later? -- like we generally use the first year we added a file.